scripts/install_cli.py

```diff
@@ -472,15 +472,21 @@ def download(url, filePath):
 def createSymlink():
     if is_win:
         link = os.path.join(getResourcesFolder(), 'wakatime-cli.exe')
-        if os.path.exists(link):
+        exists = os.path.exists(link)
+        if exists and not os.path.islink(link):
             try:
                 os.remove(link)
+                exists = False
             except:
                 log(traceback.format_exc())
-        try:
-            shutil.copy2(getCliLocation(), link)
-        except:
-            log(traceback.format_exc())
+        if not exists:
```

Comment:
I'll change this to try copying even when already exists, but only if we've successfully downloaded an updated wakatime-cli.

Reply:
Added to master with 9c1c0b5d7407c67d159e9d9bc9b886947ef9cf00 and bbfa0fb9a31fe812722468d7000a898bb83fc2fe.